javascript/new-src/src/factories/ola.js

```diff
@@ -22,6 +22,29 @@
 ola.factory('$ola', ['$http', '$window', 'OLA',
   function($http, $window, OLA) {
     'use strict';
+    // holds the highest channel that was used by faders or the keypad
+    var highestChannelNumberUsed = 0;
+
+    // Search for the highest channel in the array `dmx`
+    // that having a value greater than MIN_CHANNEL_VALUE
```

Comment:
Grammar wise we want either:
```suggestion
    // that has a value greater than MIN_CHANNEL_VALUE
```
Or:
```suggestion
    // having a value greater than MIN_CHANNEL_VALUE
```